coalib/output/ConsoleInteraction.py

```diff
@@ -316,7 +316,7 @@ def print_affected_files(console_printer,
                          file_dict,
                          color=True):
```

Comment:
commit message not good enough, shortlog should be

ConsoleInteraction: Fix ``afected``->``affected``

also, body is redundant here at all.

also, does this fix any existing open issue there is already? if so, please link it using "Closes issue_url"